packages/ethereum/contracts/HoprChannels.sol

```diff
@@ -125,7 +125,7 @@ contract HoprChannels is IERC777Recipient, ERC1820Implementer {
         bytes32 nextCommitment,
         uint256 ticketEpoch,
         uint256 ticketIndex,
-        bytes32 proofOfRelaySecret,
+        bytes32 porSecret,
```

Comment:
please unabbreviate - it was clearer before.

Reply:
changed